### PR TITLE
Docs Changes

### PR DESCRIPTION
**Pull request by YouQam**

### CODE REVIEW

**Comment by nuest** [comment]
@YouQam No need to worry about existing commit messages, but this is worth a look for you: https://github.blog/2013-01-22-closing-issues-via-commit-messages/

You can still add "closes #43" to the first PR comment above and that should work.

File: tests/test_cli.py [hunk 1]
```diff
@@ -32,88 +33,94 @@ def test_geojson_invalid_second_input(script_runner):
...
         '-b',
-        'tests/testdata//geojson/muenster_ring_zeit.geojson')
+        '-input=', 'tests/testdata//geojson/muenster_ring_zeit.geojson')
     assert ret.success, "process should return success"
     assert "[7.60168075561523, 51.9488147720619, 7.64725685119629, 51.9746240298775]" in ret.stdout, "bbox is printed to console"
 
 def test_geojson_bbox_long_name(script_runner):
     ret = script_runner.run('geoextent',
-        '--bounding-box', 'tests/testdata//geojson/muenster_ring_zeit.geojson')
+        '--bounding-box', '-input=', 'tests/testdata//geojson/muenster_ring_zeit.geojson')
```
Comment (nuest) [inline tests/test_cli.py]: Is the `-input=` needed now? It should not be, so I suggest to leave some of the tests without it.

File: docs/source/howto/api.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Takes raw data (in the form of a string, boolean, boolean) to output result based on these fields.
+
+**Parameters:**   
+   - **input** - a string value of input file or path    
+   - **bbox** - a boolean value to extract spatial extent (bounding box)
+   - **time** - a boolean value to extract temporal extent
+
+Examples:-
+----------
+**Ex (1):** Extracting bounding box:
```
[the remark this answers is not in the document]
Comment (YouQam) [reply docs/source/howto/api.rst]: https://github.com/YouQam/geoextent/commit/2c120b7c44ccd203a83ca75f78447c8c3e3cddcd

File: docs/source/howto/cli.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+   :prog: \
+
+
+Examples:-
+----------
+**Ex (1):** Show help message:
+
+::
+
+   python3 geoextent -h
```
[the remark this answers is not in the document]
Comment (YouQam) [reply docs/source/howto/cli.rst]: It works with out `python3`. I will remove it and add a note.

[the remark this answers is not in the document]
Comment (YouQam) [reply docs/source/howto/cli.rst]: https://github.com/YouQam/geoextent/commit/710c843dacc8705979f491d7ea4f61480c993b00

Screenshot from 2020-03-19 00-49-54

File: docs/source/install.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
+
+Installing geoextent
+====================
+
+geoextent requires Python 3.x and above on Linux.
```
[the remark this answers is not in the document]
Comment (YouQam) [reply docs/source/install.rst]: https://github.com/YouQam/geoextent/commit/d5b631adb0a291ad34f44bdd8a1ba1f63c21efaf

File: tests/test_cli.py (hunk 1 above)
Comment (YouQam) [reply tests/test_cli.py]: Yes, it is needed.

Comment (YouQam) [reply tests/test_cli.py]: If I update it to previous impl. the `-formats` won't work and going to raise an error.

File: docs/source/howto/cli.rst [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+   :hide-code:
+
+   import geoextent.lib.extent as geoextent
+   geoextent.fromFile('../tests/testdata/geojson/muenster_ring_zeit.geojson', False, True)
+
+**Ex (4):** Extract both bounding box and time interval:
+
+::
+
+   python3 geoextent -b -t 'file.geojson'
```
[the remark this answers is not in the document]
Comment (YouQam) [reply docs/source/howto/cli.rst]: untitled(1)

[the remark this answers is not in the document]
Comment (YouQam) [reply docs/source/howto/cli.rst]: https://github.com/YouQam/geoextent/commit/a9ca739287a311c92a98e7f79907ea8cc900d802

**Pull request merged**